LICENSE

```diff
@@ -211,3 +211,96 @@ Home page: https://github.com/gradle/gradle
 License: https://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
+
+This product includes code from Apache Iceberg.
+
+* spec/rest-catalog-open-api.yaml
+* integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisRestCatalogIntegrationTest.java
+
+Copyright: Copyright 2017-2025 The Apache Software Foundation
+Home page: https://iceberg.apache.org
+License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes code from OpenAPITool openapi-generator
+
+* server-templates/formParams.mustache
+* server-templates/apiService.mustache
+* server-templates/bodyParams.mustache
+* server-templates/pojo.mustache
+* server-templates/headerParams.mustache
+* server-templates/apiServiceImpl.mustache
+* server-templates/queryParams.mustache
+* server-templates/api.mustache
+
+Copyright: Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech) Copyright 2018 SmartBear Software
+Home page: https://openapi-generator.tech/
+License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes code from Google Docsy.
```

Comment:
If this is going to be removed in the next release, why document it rather than removing it before 0.9.0?

Reply:
I think that it's good to have the doc in the source distribution (it's very valuable for users). I would propose to keep it for now, and remove from `LICENSE` *if* we remove it later from the source distribution.

@rdblue wdyt ?

Reply:
@rdblue thoughts ?

Reply:
I don't think it is a blocker as long as the code is documented.

Reply:
Fair. Let's include this for now (I will discuss with the community after the release).